### ISSUE TITLE
Local notifications do not work if UseNotifications() is registered without delegate

### ISSUE THREAD

**Issue by suppenburger** [shinyorg/shiny#324]
# IF YOU DON'T ANSWER THIS TEMPLATE - THE BOT WILL AUTOMATICALLY CLOSE YOUR ISSUE!  PLEASE DON'T ASK FOR WORKAROUNDS OR TIMELINES - THIS IS FREE SOURCE CODE

## Please check all of the platforms you are having the issue on (if platform is not listed, it is not supported)

 - [X] iOS
 - [ ] Android
 - [ ] UWP

 PLATFORMS THAT YOU CANNOT FILE ISSUES ON RIGHT NOW: Tizen, watchOS, tvOS

## Feature Library (beta components are not fully done - issues MUST include full reproducible sample and stacktrace or they will be closed without explanation!!!)

 - [X] Core
 - [ ] Locations
 - [ ] HTTP Transfers
 - [X] Notifications
 - [ ] Sensors
 - [ ] AppCenter Logging Integration
 - [ ] Location Sync (beta)
 - [ ] BluetoothLE (beta)
 - [ ] Beacons (beta)
 - [ ] Push (beta)
 - [ ] SQLite Integration (beta)

## Version of Library
Shiny.Notifications" Version="1.1.0.1570"

## Steps to Reproduce
If register notifications with
```
services.UseNotifications(true);
```
local notifications work only if app is not running in foreground.

If register notifications with a delegate
```
NotificationCategory[] notificationCategories = { };
services.UseNotifications<NotificationService>(true, notificationCategories);
```
local notifications work both in foreground and background.

If testing with the samples project it shows that notification which are registered without a delegate show a pop-up windows instead of a native iOS notification. This behavior could be related to the issue.

This issue might report the same problem
https://github.com/shinyorg/shiny/issues/260

**Comment by suppenburger** [comment]
Not sure why the bot closed it. Behavior can be reproduced with the official samples project.

**Comment by aritchie** [comment]
The bot is broken at the moment.  The issue is fixed for 1.2
